tasks/install.yml

```diff
@@ -44,7 +37,7 @@
     home: "{{ redis_install_dir }}"
     shell: /bin/false
     system: yes
-  when: user_exists|failed
+    state: present
```

Comment:
There is a risk here of setting the shell to `/bin/false` of a login user, if for any reason `redis_user` is set to anything else but `redis`